kusd/validator/validator.go

```diff
@@ -49,11 +49,13 @@ type Validator interface {
 	AddProposal(proposal *types.Proposal) error
 	AddVote(vote *types.Vote) error
 	AddBlockFragment(blockNumber *big.Int, round uint64, fragment *types.BlockFragment) error
+	Deposits() ([]*types.Deposit, error)
```

Comment:
we are prob going to need to split this interface, just thinking how and where

Reply:
:)